third_party/terraform/tests/resource_spanner_database_test.go

```diff
@@ -26,6 +26,17 @@ func TestAccSpannerDatabase_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet("google_spanner_database.basic", "state"),
 				),
 			},
+			{
+				// Test import with default Terraform ID
+				ResourceName: "google_spanner_database.basic",
+				ImportState:  true,
```

Comment:
This should also have an `ImportStateVerify: true` (I honestly don't know why that isn't just built in, we've accidentally forgotten to set it so many times and there's not much of a reason to import and not verify)

Also, the test is failing because we don't set `ddl` on read. I think that instead of setting it on read in this PR and making it too big, we should just ignore it in the test (we already do this for the generated tests). You can do that by setting the line:
`ImportStateVerifyIgnore: []string{"ddl"},`
You might have to do that on all the import steps.